src/bootstrap/doc.rs

```diff
@@ -60,12 +73,12 @@ macro_rules! book {
 // NOTE: When adding a book here, make sure to ALSO build the book by
 // adding a build step in `src/bootstrap/builder.rs`!
```

Comment:
Can we add a note here about making sure submodule is added when populating these? I wonder if it's possible to automate the submodule detection -- in theory, we can at startup or otherwise have a list of submodules and do a prefix search or so (e.g., src/tools/cargo/src/doc will note that src/tools/cargo is a prefix and a submodule)...

Probably just leaving a comment is OK for now. We don't really add submodules much these days.